gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
I was asking myself the same thing, but the example shows that you are supposed to do stuff like this and *we* are doing something wrong (what? us? mistakes? impossible!)